website/docs/prepare-for-merge.md

```diff
@@ -0,0 +1,259 @@
+---
+id: prepare-for-merge
+title: Prepare for The Merge
+sidebar_label: Prepare for The Merge
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[The Merge](https://ethereum.org/en/upgrades/merge/) will fully transition Ethereum's consensus mechanism from proof-of-work to proof-of-stake. This is made possible by the [Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/), a new Ethereum network layer that implements proof-of-stake consensus. After The Merge, this consensus layer will be fully "merged" with Ethereum's execution layer, and miners will be fully replaced by validators on Ethereum Mainnet.
+
+This guide will walk you through the changes that you need to make in preparation for The Merge.
+
+:::info New user?
+
+This guidance is targeted at users who are already running Prysm. If you're starting from scratch, our [Quickstart](./install/install-with-script.md) will help you configure a Merge-ready node from start to finish.
+
+:::
+
+## The Merge: Before and after
+
+| Before The Merge                                                                                     | After The Merge                                                                                       |
+|------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
+| You don't need to run a local execution client. You can use a service like Infura instead.           | You **do** need to run an execution client. You **can't** use a service like Infura.                  |
+| The HTTP connection between beacon node and execution node doesn't need to be authenticated via JWT. | The HTTP connection between beacon node and execution node **does** need to be authenticated via JWT. |
+| Execution clients don't need to specify a TTD value.                                                 | Execution clients **do** need to specify a TTD value.                                                 |
+| Miners receive transaction fees.                                                                     | **Validators** receive transaction fees.                                                              |
+| A 1TB hard drive is enough.                                                                          | A **2TB+ SSD** is required.                                                                           |
+
+
+<br />
+
+
+Let's step through each of these changes. 
+
+
+## Generate JWT token
+
+A **JWT token** will allow your beacon node to form an authenticated HTTP connection with your execution node's engine API, a new API that facilitates The Merge. Although Ropsten and Sepolia are the only networks that currently require authenticated HTTP, it will soon be required on Goerli-Prater and Mainnet. We recommend doing this now regardless of the network you're running on. 
```

Comment:
This has too many details that aren't relevant to users. We should make this section as simple as possible and say that Prysm needs to authenticate its connection to its execution node. There are several methods to generate these credentials: x, y, z. We should leave it at that